lightning/src/ln/channelmanager.rs

```diff
@@ -8842,7 +8862,7 @@ where
 		&self, offer: &Offer, quantity: Option<u64>, amount_msats: Option<u64>,
 		payer_note: Option<String>, payment_id: PaymentId, retry_strategy: Retry,
 		max_total_routing_fee_msat: Option<u64>
-	) -> Result<(), Bolt12SemanticError> {
+	) -> Result<(), Bolt12CreationError> {
```

Comment:
Could you update the other BOLT12-related `ChannelManager` methods to use this error, too? Ideally, change one per commit.